eng/build.yml

```diff
@@ -58,6 +58,14 @@ jobs:
       ${{ if and(eq(parameters.osGroup, 'MacOS'), eq(variables['System.TeamProject'], 'public')) }}:
         vmImage: macOS-10.14
 
+      # Public OSX Xcode 12 Build Pool
+      ${{ if and(eq(parameters.osGroup, 'MacOS_cross'), ne(variables['System.TeamProject'], 'public')) }}:
+        name: Hosted Mac Internal
+
```

Comment:
I'd add the `vmImage` line here too, given you need the OSX 10.15 image for sure.

Reply:
I didn't know I could specify the queue name and the vmimage.  That means I can run internal builds too.